lib/zold/upgrades.rb

```diff
@@ -2,6 +2,26 @@
 
 module Zold
   # Class to manage data upgrades (when zold itself upgrades).
+  #
+  # This class will write the version file to the zoldata directory.
+  # The version file is a text file containing nothing but the version
+  # of the data and a newline. It is named `version`, no extension.
+  #
+  # If the data is up-to-date, the version of the data is equal to
+  # Zold::VERSION.
+  #
+  # If the version is lower than Zold::VERSION, upgrade scripts from
+  # `upgrades/` have to run. They are named `<version>.rb`, so for
+  # instance `upgrades/0.0.1.rb` etc.
+  #
+  # Only the scripts from the data version up need to run.
```

Comment:
@botanicus greater than the data version? Or >=?

Reply:
If we're on 0.1 and want to migrate to 0.2, we should run `0.2.rb`, `0.1.rb` run already when we upgraded to 0.1, I'd say. So I think this is correct.

Reply:
@botanicus I have a hard time understanding it, and one of my mates too. Could you please change it in the docs?